docks/op_expand.go

```diff
@@ -261,17 +337,36 @@ func (op *ExpandOp) forwardHandler(_ context.Context) error {
 }
 
 func (op *ExpandOp) backwardHandler(_ context.Context) error {
+	// Setup micro task done signal.
+	microTaskDone := func() {}
+	defer microTaskDone()
+
```

Comment:
What's that for?

Reply:
Failsafe for when the goroutine crashes.